cmd/manager/main.go

```diff
@@ -126,13 +128,20 @@ func main() {
 	// work
 	stopCh := signals.SetupSignalHandler()
 
+	notClient, err := client.New(cfg, client.Options{})
```

Comment:
@liamwazherealso Do you think we want to keep this var name as `notClient` :slightly_smiling_face:

Reply:
I like the ring to it. However, I will refactor.